docs/howto/create-a-release.rst

```diff
@@ -102,6 +102,115 @@ Upload build to PyPI
 
 Check PyPI for your upload.
 
+Upload to Conda
+===============
+
+NOTE: the GitHub and PYPI releases are prerequisites for releasing to
+Conda Forge!
+
+The Conda Forge release process is mostly automated by bots. These bots
+are set up during the initial project bootstrap, which has been done for
+all the projects that are part of Cylc 8.
+
+As there are inter-dependencies amongst the Cylc 8 parts, you should
+know the dependency tree, and start by the leaf nodes (i.e. a module
+A may have multiple dependants, but no dependency to other modules).
+
+For example, Cylc UI Server depends on Cylc Flow. So unless you are
+releasing only Cylc UI Server, you should release Cylc Flow first.
+
+* On GitHub, navigate to the project repository on GitHub, e.g.
+
+- https://github.com/conda-forge/metomi-isodatetime-feedstock
+- https://github.com/conda-forge/cylc-uiserver-feedstock
+- https://github.com/conda-forge/cylc-ui-feedstock
+- https://github.com/conda-forge/cylc-flow-feedstock
+- https://github.com/conda-forge/cylc-feedstock
+
+On each of these repositories, the release process should be pretty
+much the same. Your work will be mainly (if not exclusive) on
+the ``recipe/meta.yaml`` file.
+
+This is a possible order of action:
+
+- Check open pull requests—There are bots that alert about dependencies
+(N.B. conda packages do not use PYPI for dependencies, they all use
+other conda packages as dependencies!.
+- Check if you need to update the version in the Jinja2 variable right
+at the top of the file.
+- If you updated the version, set the build number back to 0.
+- If you did not update the version, increase the build number.
+- If the conda package contains a source URL, check that it is evaluated
```

Comment:
I'm not sure I know what this step is telling me to do.
if I'm looking at:

https://github.com/conda-forge/cylc-uiserver-feedstock/blob/a7e7214c49ca420899cf3a6e7e0834cc1eff16fc/recipe/meta.yaml#L8-L10

how do I carry out this check? How do I generate the hyperlink and how do I generate the new checksum?

Reply:
>how do I carry out this check?

I do that mentally or with a notepad. The `meta.yaml` is parsed first by a Jinja processor. If you look at the top of the file, there are variables declared (we are using the template from Conda Forge, so 99% of other recipes in Conda Forge should look similar).

The way I do it, is opening a notepad, pasting that URL you linked above, then manually replacing the variables (a manual `eval` ? :smiley: ). Then paste it into the browser, and confirm I don't get a 404. I added this step because it happened once to me that I had a wrong value, and PYPI returned 404. Finding that later wasn't really easy.

I've changed the text to 

>- If the conda package contains a source URL, check that it is evaluated
correctly using the Jinja variables, and update the ``sha256`` value.

WDYT?

>How do I generate the hyperlink and how do I generate the new checksum?

To be honest, I never do that for the Python dependencies. I simply go to https://pypi.org, search for something like `metomi-isodatetime`, then click on "Download Files". At the right side, there's a button to show the hashes, which includes the value used here.

Alternatively, I think the release-manager dev could prefer to download the file and run something like `sha256sum $file`.

I've added the following now to the end of the text

>- If the conda package contains a source URL, check that it is evaluated
correctly using the Jinja variables, and update the ``sha256`` value
(you can get this value from PYPI, or use some tool like ``sha256sum``).

Reply:
> If the conda package contains a source URL, check that it is evaluated
correctly using the Jinja variables, and update the `sha256` value.

suggestion: 
>  If the conda package contains a source URL, check that it is evaluated
correctly by manually replacing the Jinja variables, and trying this out in a browser. 
Then update the `sha256` value. (you can get this value from PYPI, or use some 
tool like sha256sum).

Reply:
Suggestion applied! :tada: thanks Tim!